lms/static/scripts/bootstrap.js

```diff
@@ -7,6 +8,9 @@ sinon.assert.expose(assert, { prefix: null });
 
 // Configure Enzyme for UI tests.
 configure({ adapter: new Adapter() });
+afterEach(() => {
+  unmountAll();
+});
```

Comment:
Not sure if we have a tests bootstrap/init/set-up script in other projects, but if we do, we should do this in all of them, so that we don't have to remember unmounting every time.

Reply:
I believe we have a `bootstrap.js` file in every project to set up Sinon, asserts, the Enzyme adapter etc.